py/mpconfig.h

```diff
@@ -1372,6 +1372,10 @@ typedef double mp_float_t;
 #define MICROPY_PY_SYS_EXC_INFO (0)
 #endif
 
+#ifndef MICROPY_PY_SYS_EXECUTABLE
```

Comment:
A comment here similar to other options would be helpful.

Reply:
Done!